src/PresentationalComponents/Cards/OverviewDashbarCard/__tests__/OverviewDashbarCard.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import { OverviewDashbarCard } from '../OverviewDashbarCard';
+import {
+  PATHWAYS,
+  INCIDENTS,
+  CRITICAL_RECOMMENDATIONS,
+  IMPORTANT_RECOMMENDATIONS,
+} from '../../../../AppConstants';
+
+// this component returns the appropriate Card component based on the title it receives
+describe('OverviewDashbarCard', () => {
+  const user = userEvent.setup();
+
+  it('Should render Pathways Card', async () => {
+    const onClickFilterByTitle = jest.fn();
+    render(
+      <OverviewDashbarCard
+        title={PATHWAYS}
+        count={1}
+        onClickFilterByTitle={onClickFilterByTitle}
+      />
+    );
+    await waitFor(() => screen.getByText(PATHWAYS));
+
+    // ensure that the correct text is displayed
+    screen.getByText(PATHWAYS);
+    screen.getByText(/1/);
```

Comment:
Aren't these missing an `expect()` in order to actually assert that the text is displayed?

Reply:
You are right! thank you for noticing 
I have added them 👍🏼